### ISSUE TITLE
aria-haspopup should be on the <a> tag, not the <li> for dropdown menus

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/wvu-design-system#285]
## Steps to reproduce the issue

  1. Find a site that uses dropdowns via the Design System (like [Academic Integrity](https://academicintegrity.wvu.edu/)).
  1. Right click > Inspect Element on a top level dropdown menu in the main navigation.

## Results

Note that the `aria-haspopup` attribute is on the `<li>` element:

```html
<li class="dropdown" aria-haspopup="true">
  <a class="nav-link" href="#">About</a>
  ...
</li>
```

## Expected results

The `aria-haspopup` attribute should be applied to the first `<a>` tag within the `<li>`:

```html
<li class="dropdown">
  <a aria-haspopup="true" class="nav-link" href="#">About</a>
  ...
</li>
```

This stems from new checks in Siteimprove's NextGen accessibility platform. It flags this as an error. If you have access to wvu.edu in Siteimprove, you can [see this check](https://my2.siteimprove.com/Accessibility/41047/NextGen/Issue?ruleId=18&pageSegments=&issueKind=1&conformance=&siteGoal=) in the new platform.

Since `aria-haspopup` is applied via JavaScript, that's where it'll need to be fixed—in [responsive-nav.js](https://github.com/wvuweb/wvu-design-system/blob/master/javascripts/vendor/responsive-nav/responsive-nav.js). This may require changing the responsive nav / dropdown source to a new repo if this change would get overridden via a new `npm ci` / `npm install` / `npm update`.
